src/kixi/heimdall/group.clj

```diff
@@ -0,0 +1,15 @@
+(ns kixi.heimdall.group
+  (:require [kixi.heimdall.components.database :as db]
+            [kixi.heimdall.util :as util]
+            [qbits.alia.uuid :as uuid]))
+
+(defn create!
+  [session {:keys [owner_id] :as group}]
+  (let [group-id (uuid/random)
+        group-data (assoc group
+                          :id group-id
+                          :created (util/db-now)
```

Comment:
I wouldn't handle it like this, because this is fairly rigid for change (say we need another type of role or something).  I would create groups independently, and role is is also a table containing group_id, user_id and value of role.